pttrack/forms.py

```diff
@@ -18,38 +18,56 @@ class Meta:
 
 class ReferralFollowup(ModelForm):
     '''The form instantiation of a followup for PCP referral.'''
+
+    boolean_choices = ((True, 'Yes'),(False, 'No'))
+    PATIENT_REACHED_HELP = "If 'No'is selected, the remaining fields will not be required"
+    patient_reached = BooleanField(widget=RadioSelect(choices=boolean_choices), 
+                                                        required = False,
+                                                        help_text=PATIENT_REACHED_HELP, 
+                                                        label="Were you able to contact the patient?")
+
     class Meta:
         model = followup_models.ReferralFollowup
-        exclude = ['patient', 'author', 'author_type']
+        fields = ['contact_method',
+            'contact_resolution',
+            'patient_reached',
```

Comment:
I was thinking that we would add a boolean property to the contact resolution--if it was a wrong number or whatever we *know* whether the person was successful.